backend/homework/7/src/main/java/org/example/Main.java

```diff
@@ -0,0 +1,56 @@
+
+package org.example;
+
+import org.example.config.AppConfig;
+import org.example.entity.Vehicle;
+import org.example.functions.HighestPriceVehicle;
+import org.example.functions.LowestPriceVehicle;
+import org.example.functions.PrintDetails;
+import org.example.services.Factory1VehicleService;
+import org.example.services.Factory2VehicleService;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+
+import java.util.List;
+
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        ApplicationContext context = new AnnotationConfigApplicationContext(AppConfig.class);
+
+        Factory1VehicleService factory1VehicleService = context.getBean(Factory1VehicleService.class);
+        Factory2VehicleService factory2VehicleService = context.getBean(Factory2VehicleService.class);
+
+        List<Vehicle> factory1Vehicles = factory1VehicleService.generateVehicle(3);
```

Comment:
Use meaningful and descriptive names for your variables rather than using numerical suffixes.